SetupDataPkg/ConfApp/SetupConf.c

```diff
@@ -599,59 +496,42 @@ ProcessDfciUsbInput (
 {
   EFI_STATUS  Status;
   CHAR16      *FileName;
-  CHAR16      *FileName2;
   CHAR8       *XmlString;
   UINTN       XmlStringSize;
 
-  DfciUiExitSecurityBoundary ();
-
   FileName  = NULL;
   XmlString = NULL;
 
   //
-  // Process request
+  // Process request, from the DEFAULT_USB_FILE_NAME
   //
-  Status = BuildUsbRequest (L".svd", &FileName);
-  if (EFI_ERROR (Status)) {
-    DEBUG ((DEBUG_ERROR, "Error building Usb Request. Code=%r\n", Status));
-  } else {
+  FileName = AllocateCopyPool (sizeof (DEFAULT_USB_FILE_NAME), DEFAULT_USB_FILE_NAME);
+  if (NULL != FileName) {
     Status = DfciRequestJsonFromUSB (
                FileName,
                &XmlString,
                &XmlStringSize
                );
     if (EFI_ERROR (Status)) {
-      FileName2 = AllocateCopyPool (sizeof (DEFAULT_USB_FILE_NAME), DEFAULT_USB_FILE_NAME);
-      if (NULL != FileName2) {
-        Status = DfciRequestJsonFromUSB (
-                   FileName2,
-                   &XmlString,
-                   &XmlStringSize
-                   );
-        if (EFI_ERROR (Status)) {
-          DEBUG ((DEBUG_ERROR, "Error loading backup file\n"));
-          FreePool (FileName2);
-        } else {
-          FreePool (FileName);
-          FileName = FileName2;
-        }
-      }
+      DEBUG ((DEBUG_ERROR, "Error loading backup file\n"));
+      FreePool (FileName);
+      FileName = NULL;
     }
+  }
 
-    if (EFI_ERROR (Status)) {
-      DEBUG ((DEBUG_ERROR, "Error processing Dfci Usb Request. Code=%r\n", Status));
-    } else {
-      DEBUG ((DEBUG_INFO, "DfciUsb Request processed normally\n"));
-      // The XmlStringSize returned will be NULL terminated, thus trim off the tail.
-      Status = ApplySettings (XmlString, XmlStringSize - 1);
-      if (Status == EFI_MEDIA_CHANGED) {
-        // MEDIA_CHANGED is a good return, It means that a JSON element updated a mailbox.
-        Status = EFI_SUCCESS;
-      }
+  if (EFI_ERROR (Status)) {
+    DEBUG ((DEBUG_ERROR, "Error processing Dfci Usb Request. Code=%r\n", Status));
+  } else {
+    DEBUG ((DEBUG_INFO, "DfciUsb Request processed normally\n"));

```

Comment:
Should these messages not say DFCI?

Reply:
This is gone. The DFCI XML related branding are replaced with SVD XML.